Flabbergasted-Flamingos/bangazon-api
# added product view in orders

# Description
Add description about what this pull request covers.

## Number of Fixes
Type of number of fixes included in this PR.

## Related Ticket(s)
List related ticket(s) that apply to the work done in this pull request

## Problem to Solve
Describe the problem(s) this pull request solves.

## Proposed Changes
Describe the proposed changes.

## Expected Behavior
Describe the expected behavior.

## Steps to Test Solution
```
git fetch --all
git checkout JDB_faker
rm -rf api/migrations
rm db.sqlite3
python manage.py makemigrations api
python manage.py migrate
python manage.py seed api --number=15
python manage.py runserver
```
Run all of that and check the api view.
Go to orders and check to see if the products are nested.

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.
